### ISSUE TITLE
IR Analysis SQL throws an error in Utilities tab

### ISSUE THREAD

**Issue by chrisknoll** [OHDSI/Atlas#2875]
### Expected behavior
You should be able to see the analysis SQL for a given design.

### Actual behavior
The SQL view under the Utilities tab of Incidence Rates fails to render due to error.

### Steps to reproduce behavior

1. Create a new Analysis.
2. Add a few target and outcome cohorts
3. Save and switch over to the Utilities tab and show SQL.
4. Observe error:
```
knockout-latest.js:16 Uncaught Error: The language "sql" has no grammar.
    at Object.highlight (prism.js:668:11)
    at highlightJS (HighLightUtils.js:8:17)
    at new SqlExportPanel (sqlExportPanel.js:37:33)
    at Object.createViewModel (CommonUtils.js:24:17)
    at d (knockout-latest.js:85:495)
    at knockout-latest.js:86:460
    at knockout-latest.js:79:19
    at b (knockout-latest.js:33:140)
    at MutationObserver.c (knockout-latest.js:33:178)
````

**Comment by chrisknoll** [comment]
Will be fixed with #2877.

**Comment by anthonysena** [comment]
Closed via #2877
